src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+public class Ball extends Lottery {
```

Comment:
Ball extends Lottery - невірний підход. Коли у нас відношення між об'єктами **А** extends **В**, то це означає що А **являється** В. Наприклад Dog extends Animal - собака являється твариною. Або Літак являється Транспортом. Тверждження, що М'яч являється Лотереєю очевидно невірне. 
```suggestion
public class Ball {
```